airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -128,6 +130,7 @@ public DefaultReplicationWorker(final String jobId,
     this.recordSchemaValidator = recordSchemaValidator;
     this.metricReporter = metricReporter;
     this.connectorConfigUpdater = connectorConfigUpdater;
+    this.featureFlagClient = featureFlagClient;
```

Comment:
This doesn't seemed used. Are we planning to use this?

Reply:
I had removed this, but it causes a conflict with `origin/master`, so I left it in to resolve that conflict.  Will remove it again.